.circleci/config-2_1.yml.mid_res.patch

```diff
@@ -1,8 +1,4 @@
-diff --git .circleci/config-2_1.yml .circleci/config-2_1.yml
```

Comment:
I have the feeling that something went wrong during patch creation.
If you try git apply check you get an error:
error: patch fragment without header at line 1: @@ -19,39 +19,51 @@ default_env_vars: &default_env_vars

Reply:
As per readme and sh you don't use `git apply` but `patch -o config-2_1.yml.HIGHRES config-2_1.yml config-2_1.yml.high_res.patch` i.e which works perfectly for me.

Reply:
yes, of course, my bad here. Sorry for the confusion

Reply:
np, this patch, diff, apply, change etc is very fiddly :-)